multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/FileUploadThreadPoolConfiguration.java

```diff
@@ -0,0 +1,158 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import java.util.Comparator;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.PriorityBlockingQueue;
+import java.util.concurrent.RunnableFuture;
+import java.util.concurrent.SynchronousQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import javax.inject.Inject;
+
+import org.cloudfoundry.multiapps.controller.core.util.ApplicationConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FileUploadThreadPoolConfiguration {
```

Comment:
There are 2 internal classes and 1 enum here, maybe it'll be better if they are outside.

Reply:
refactored as separate classes